run/test/specs/message_checks_groups_android.spec.ts

```diff
@@ -0,0 +1,546 @@
+  });
+  await device1.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Message body",
+    text: replyMessage,
+  });
+  await device3.clickOnElementAll({
+    strategy: "id",
+    selector: `network.loki.messenger:id/scrollToBottomButton`,
+  });
+  await device3.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: "Message body",
+    text: replyMessage,
+  });
+  // Close app
+  await closeApp(device1, device2, device3);
+}
+/* COMMENTED OUT UNTIL BUG FOR SCROLL TO BOTTOM IS FIXED */
+
+async function sendLongMessageGroup(platform: SupportedPlatformsType) {
+  const testGroupName = "Message checks for groups";
+  const { device1, device2, device3 } = await openAppThreeDevices(platform);
+  // Create users A, B and C
+  const [userA, userB, userC] = await Promise.all([
+    newUser(device1, "Alice", platform),
+    newUser(device2, "Bob", platform),
+    newUser(device3, "Charlie", platform),
+  ]);
+  // Create contact between User A and User B
+  await createGroup(
+    platform,
+    device1,
+    userA,
+    device2,
+    userB,
+    device3,
+    userC,
+    testGroupName
+  );
+  const longText =
+    "Mauris sapien dui, sagittis et fringilla eget, tincidunt vel mauris. Mauris bibendum quis ipsum ac pulvinar. Integer semper elit vitae placerat efficitur. Quisque blandit scelerisque orci, a fringilla dui. In a sollicitudin tortor. Vivamus consequat sollicitudin felis, nec pretium dolor bibendum sit amet. Integer non congue risus, id imperdiet diam. Proin elementum enim at felis commodo semper. Pellentesque magna magna, laoreet nec hendrerit in, suscipit sit amet risus. Nulla et imperdiet massa. Donec commodo felis quis arcu dignissim lobortis. Praesent nec fringilla felis, ut pharetra sapien. Donec ac dignissim nisi, non lobortis justo. Nulla congue velit nec sodales bibendum. Nullam feugiat, mauris ac consequat posuere, eros sem dignissim nulla, ac convallis dolor sem rhoncus dolor. Cras ut luctus risus, quis viverra mauris.";
+  // Sending a long text message
+  const replyMessage = `${userA.userName} message reply`;
+  await device1.inputText("accessibility id", "Message input box", longText);
+  // Click send
+  await device1.clickOnElement("Send message button");
+  await device1.waitForTextElementToBePresent({
+    strategy: "accessibility id",
+    selector: `Message sent status: Sent`,
+    maxWait: 50000,
+  });
+
+  await Promise.all([
+    device2.waitForTextElementToBePresent({
+      strategy: "accessibility id",
+      selector: "Message body",
+      text: longText,
+    }),
+    device3.waitForTextElementToBePresent({
+      strategy: "accessibility id",
+      selector: "Message body",
+      text: longText,
+    }),
+  ]);
+  await sleepFor(1000);
+  await device1.clickOnElementAll({
+    strategy: "id",
+    selector: `network.loki.messenger:id/scrollToBottomButton`,
+  });
+  await device2.replyToMessage(userA, longText);
+  await sleepFor(1000);
```

Comment:
can we waitFor something?

Reply:
same as above, removed sleepFor and replaced with a maxwait in the clickOnElementAll functions